index.html

```diff
@@ -311,72 +330,163 @@ <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
       };
       </pre>
       <p class="note">To keep the performance overhead to minimum the
-      application should only subscribe to event types that it is interested
+      application ought to only subscribe to event types that it is interested
       in, and disconnect the observer once it no longer needs to observe the
       performance data. Filtering by name is not supported, as it would
       implicitly require a subscription for all event types — this is possible,
       but discouraged, as it will generate a significant volume of events.</p>
       <section>
         <h2><dfn>observe()</dfn> method</h2>
         <p>The <a>observe()</a> method instructs the user agent to
-        <dfn>register the observer</dfn> and must run these steps:</p>
+        <dfn>register the <var>observer</var></dfn> and must run these steps:</p>
```

Comment:
For both observer and options, I prefer "Let observer be..." style where the variables are explicitly defined

Reply:
Also, observer here is used both as a var and as a member of a potential PO. Can you pick a different name for the local variable?

Reply:
Done